app/controllers/application_controller.rb

```diff
@@ -84,6 +84,10 @@ def set_proposal_votes(proposals)
       @proposal_votes = current_user ? current_user.proposal_votes(proposals) : {}
     end
 
+    def set_legislation_question_votes(questions)
+      @legislation_question_votes = current_user ? current_user.legislation_question_votes(questions) : {}
+    end
```

Comment:
Since this method is only used in one controller, how about moving it to that controller?

I know there are similar methods in the `ApplicationController`, but there are also places where it's done a different way. For instance, the `Budgets::InvestmentsController` uses the `load_investment_votes` method, which is defined there.